deployment/ansible/roles/model-my-watershed.geoprocessing/templates/upstart-akka-http.conf.j2

```diff
@@ -0,0 +1,13 @@
+description "Akka HTTP Server"
+
+start on runlevel [3]
+stop on shutdown
+
+kill timeout 20
+kill signal CONT
+respawn
+
+expect fork
```

Comment:
Can we get rid of this stanza if we drop the trailing `&` in the `exec` command?